Cosmetics.py

```diff
@@ -28,6 +28,33 @@ def patch_dpad(rom, settings, log, symbols):
         rom.write_byte(symbols['CFG_DISPLAY_DPAD'], 0x00)
     log.display_dpad = settings.display_dpad
 
+    
+def patch_settings_string(rom, settings, log, symbols):
+    # Display D-Pad HUD
+    if settings.show_seed_info:
+        rom.write_byte(symbols['CFG_SHOW_SETTING_INFO'], 0x01)
+    else:
+        rom.write_byte(symbols['CFG_SHOW_SETTING_INFO'], 0x00)
+    line_len = 26
+    custom_msg = settings.user_message.strip()[:48]
+    if len(custom_msg) <= line_len:
+        msg = [custom_msg, ""]
+    else:
+        # try to split message
+        msg = [custom_msg[:line_len], custom_msg[line_len:]]
+        part1 = msg[0].split(' ')
+        if len(part1[-1]) + len(msg[1]) < line_len:
+            msg = [" ".join(part1[:-1]), part1[-1] + msg[1]]
+        else:
+        # Is it a URL?
+            part1 = msg[0].split('/')
+            if len(part1[-1]) + len(msg[1]) < line_len:
+                msg = ["/".join(part1[:-1]) + "/", part1[-1] + msg[1]]
+        for idx,part in enumerate(msg):
```

Comment:
This loop appears to be indented too far, so messages under the line_len don't get used.
Edit: I can't figure out how to make a suggestion on github when I've already commented.

Reply:
Unindenting this loop fixes my issue (My custom message was simply `test`).